app/models/street/StreetEdgeTable.scala

```diff
@@ -199,6 +199,48 @@ object StreetEdgeTable {
     }
   }
 
+
+  /**
+    * Calculates the distance audited today by all users
```

Comment:
Can you finish comments with a period, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)